examples/deadline/All-In-AWS-Infrastructure-Basic/python/README.md

```diff
@@ -12,66 +12,86 @@ These instructions assume that your working directory is `examples/deadline/All-
+        # unlimited Arnold licenses
+        UsageBasedLicense.for_arnold()
+    ]
+    ```
+
+    ---
+
+    **Note:** The next two steps are optional. You may skip these if you do not need SSH access into your render farm.
+
+    ---
+6.  Create an EC2 key pair to give you SSH access to the render farm:
+
+    ```bash
+    aws ec2 create-key-pair --key-name <key-name>
+    ```
 7.  Change the value of the `key_pair_name` variable in `package/config.py` to your value for `<key-name>` in the previous step: <br><br>**Note:** Save the value of the "KeyMaterial" field as a file in a secure location. This is your private key that you can use to SSH into the render farm.
-```python
-self.key_pair_name: Optional[str] = '<your key pair name>'
-```
-8. Choose the type of database you would like to deploy and change the value of the `deploy_mongo_db` variable in `package/config.py` accordingly:
-```python
-# True = MongoDB, False = Amazon DocumentDB
-self.deploy_mongo_db: bool = False
-```
-9. If you set `deploy_mongo_db` to `True`, then you must accept the [SSPL license](https://www.mongodb.com/licensing/server-side-public-license) to successfully deploy MongoDB. To do so, change the value of `accept_sspl_license` in `package/config.py`:
-```python
-self.accept_sspl_license: MongoDbSsplLicenseAcceptance = MongoDbSsplLicenseAcceptance.USER_REJECTS_SSPL
-```
+
+    ```python
+    self.key_pair_name: Optional[str] = '<your key pair name>'
+    ```
+8.  Choose the type of database you would like to deploy and change the value of the `deploy_mongo_db` variable in `package/config.py` accordingly:
+
+    ```python
+    # True = MongoDB, False = Amazon DocumentDB
+    self.deploy_mongo_db: bool = False
+    ```
+9.  If you set `deploy_mongo_db` to `True`, then you must accept the [SSPL license](https://www.mongodb.com/licensing/server-side-public-license) to successfully deploy MongoDB. To do so, change the value of `accept_sspl_license` in `package/config.py`:
+
+    ```python
+    self.accept_sspl_license: MongoDbSsplLicenseAcceptance = MongoDbSsplLicenseAcceptance.USER_REJECTS_SSPL
+    ```
 10. Stage the Docker recipes for `RenderQueue` and `UBLLicensing`:
-```bash
-# Set this value to the version of RFDK your application targets
-RFDK_VERSION=
-# Set this value to the version of AWS Thinkbox Deadline you'd like to deploy to your farm
-RFDK_DEADLINE_VERSION=
-npx --package=aws-rfdk@${RFDK_VERSION} stage-deadline \
-    --deadlineInstallerURI s3://thinkbox-installers/Deadline/${RFDK_DEADLINE_VERSION}/Linux/DeadlineClient-${RFDK_DEADLINE_VERSION}-linux-x64-installer.run \
-    --dockerRecipesURI s3://thinkbox-installers/DeadlineDocker/${RFDK_DEADLINE_VERSION}/DeadlineDocker-${RFDK_DEADLINE_VERSION}.tar.gz \
-    --output stage
-```
+
+    ```bash
+    # Set this value to the version of RFDK your application targets
+    RFDK_VERSION=
+    # Set this value to the version of AWS Thinkbox Deadline you'd like to deploy to your farm
+    RFDK_DEADLINE_VERSION=
```

Comment:
I think it would be more readable if we not left it with empty values.
Like: `RFDK_VERSION=<version_of_RFDK>` `RFDK_DEADLINE_VERSION=<version_of_deadline>`

Reply:
Good idea - applied